### ISSUE TITLE
There is no way to prevent rambox ad-related windows from reappearing on every run

### ISSUE THREAD

**Issue by eychenz** [saenzramiro/rambox#1040]
### Steps to reproduce

1. Install rambox on Fedora 26 according to the instruction.
2. Start Rambox.
3. Extra windows appear like "what do you think about rambox? share on Facebook | twitter". Welcome message appears as "welcome to 0.5.10 / view change log".
4. Close extra windows and start using Rambox.
5. Exit Rambox.
6. Start Rambox again.

### Expected behavior

See Rambox chat interfaces.

### Actual behavior

Again extra windows appear like "what do you think about rambox? share on Facebook | twitter". Welcome message appears as "welcome to 0.5.10 / view change log".
Basically, there is no way to shut those windows and prevent them from appearing again on the next run. That used to be the same case for the previous version that it was always showing messages referring to upgrade to the newer version without an option to shut it. So, everytime we had to start the chat client and press bunch of crosses to close un-related windows / popups / tooltips.

### ENVIRONMENT

> Rambox: v0.5.10

> OS: Fedora 26 workstation

> Arch: x64

**Comment by ChipWolf** [comment]
This.

**Comment by ChipWolf** [comment]
This. @saenzramiro, please give us an option to remove these irritating modals.

**Comment by saenzramiro** [comment]
We just removed those messages. Sorry guys...
